README.md

```diff
@@ -15,75 +15,72 @@
+1. Hops fits a broad need by being modular and extensible, it comes with reasonable defaults, but allows you to customize, configure, and extend almost everything when needed
 
-```shell
-npx hops init my-awesome-project
-cd my-awesome-project
-npm start
-```
+Hops is designed to simplify getting started with modern frontend tooling. It streamlines [Webpack](https://webpack.js.org) and [Jest](https://facebook.github.io/jest/) configuration featuring [Babel](https://babeljs.io) and [PostCSS](http://postcss.org).
+
+Why and how Hops is structured into modules can be found [here](./docs/modules.md) and [here is a more detailed and complete treatise on its concepts](./docs/concepts.md).
 
-Otherwise:
+### Quick start
 
 ```shell
-npm install --global hops
-hops init my-awesome-project
+npx hops init my-awesome-project
 cd my-awesome-project
 npm start
 ```
 
 This will start hops in development mode. Visit [http://localhost:8080](http://localhost:8080) to see your app in the browser and make some changes to the code in your editor to see it live-reloading.
 
-If you want to build the JS bundles for production, run `npm run build` in your terminal and to start a Node.js server just type in `npm start --production`.
+You can find a complete walk through in our [Getting Started](./docs/getting-started.md).
 
-### Project Templates
+### FAQ
 
-Hops comes with a couple of different project templates you can use with `hops init --template`. If you do not explicitly select one, `hops init` defaults to `hops-template-react`.
+#### When is Hops the framework for you?
 
-| Package                                                                                     | NPM                                                                                                                   |
-| ------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
-| [Hops Template Minimal](https://github.com/xing/hops/tree/master/packages/template-minimal) | [![npm](https://img.shields.io/npm/v/hops-template-minimal.svg)](https://www.npmjs.com/package/hops-template-minimal) |
-| [Hops Template React](https://github.com/xing/hops/tree/master/packages/template-react)     | [![npm](https://img.shields.io/npm/v/hops-template-react.svg)](https://www.npmjs.com/package/hops-template-react)     |
-| [Hops Template GraphQL](https://github.com/xing/hops/tree/master/packages/template-graphql) | [![npm](https://img.shields.io/npm/v/hops-template-graphql.svg)](https://www.npmjs.com/package/hops-template-graphql) |
+When you want to share a consistent setting over more than one project. Either now or in the future. This applies to companies with more than one development team or when you constantly set up projects that should be consistent, e.g. training projects, tutorials, consultant work, etc.
```

Comment:
Even without striving for consistency across multiple projects I may want to use hops for a (single) new project when I want more customisability than CRA gives me. So that maybe could be a second scenario in this section.

Reply:
CRA probably is not the alternative for professional users, rather writing/copying webpack configs of your own. At least that's what I see.

Reply:
Correct. But I think that’s the same target group: I don’t want to write my own webpack config again so CRA is somehow appealing, and on the other hand I do want to able to customize stuff.

Reply:
Yes, good point. Will add.